system-tests/project-fixtures/react/cypress-vite-default.config.ts

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'cypress'
+
+import * as vite from 'vite'
+
+declare global {
+  namespace Cypress {
+    interface DefineDevServerConfig {
+      viteConfig?: vite.UserConfig
+    }
+  }
+}
+
+export default defineConfig({
+  videoCompression: false, // turn off video compression for CI
```

Comment:
Probably not needed since it's off by default.

```suggestion
export default defineConfig({
```